src/MongoDB.Driver/FindOneAndUpdateOptions.cs

```diff
@@ -77,6 +78,18 @@ public Collation Collation
             set { _collation = value; }
         }
 
+        /// <summary>
+        /// Gets or sets the hint.
+        /// </summary>
+        /// <value>
```

Comment:
this class doesn't have `<value>` xml comment in other properties

Reply:
Done.